src/components/ArticlePage.jsx

```diff
@@ -73,13 +83,20 @@ const ArticlePage = () => {
       </div>
 
       <h3>Comments</h3>
+      <p className="logged-in-as">logged in as {currentUser}</p>
       <CommentForm
         articleId={articleId}
+        currentUser={currentUser}
         onCommentSubmit={handleCommentSubmit}
       />
       {comments.length > 0 ? (
         comments.map((comment) => (
-          <CommentCard key={comment.comment_id} comment={comment} />
+          <CommentCard
+            key={comment.comment_id}
+            comment={comment}
+            currentUser={currentUser}
+            onDelete={handleDeleteComment}
```

Comment:
Changing the name from handleDeleteComment to onDelete may confuse things. Try to keep it the same everywhere.